script/commands/warp.ts

```diff
@@ -90,7 +96,12 @@ function checkConfigType<
   return config.type === tokenType && config.mode === tokenMode;
 }
 
-async function handleCreate(configFile: string) {
+async function handleCreate(configFile: string, cmd: Command) {
+  type Option = {
+    ismAddress?: string;
```

Comment:
do you have to do the 0x for this one as well?

Reply:
i don't see an error but I don't think it'd hurt!

Reply:
added, merging